src/components/Capture/Document.js

```diff
@@ -59,22 +61,39 @@ class Document extends Component {
     } = this.props
     const copyNamespace = `capture.${isPoA ? poaDocumentType : documentType}.${side}`
     const title = translate(`${copyNamespace}.title`)
-    const moreProps = { ...this.props, onError: this.handleError }
-
-    return useWebcam && hasCamera ?
-      <DocumentAutoCapture
-        { ...moreProps }
-        renderTitle={ <PageTitle { ...{ title, subTitle } } smaller /> }
-        renderFallback={ isDesktop ? this.renderCrossDeviceFallback : this.renderUploadFallback }
-        containerClassName={ style.documentContainer }
-        onValidCapture={ this.handleCapture }
-      /> :
+    const propsWithErrorHandling = { ...this.props, onError: this.handleError }
+    const renderTitle = <PageTitle {...{title, subTitle}} smaller />
+    const renderFallback = isDesktop ? this.renderCrossDeviceFallback : this.renderUploadFallback
+    if (useWebcam && hasCamera) {
+      return (
+        <DocumentAutoCapture
+          {...propsWithErrorHandling}
+          renderTitle={ renderTitle }
+          renderFallback={renderFallback}
```

Comment:
minor: seems like all other values have spaces between the braces except `renderFallback` and `containerClassName` here

Reply:
Good spot. Will clean that up.